### ISSUE TITLE
Buddy discussion

### ISSUE THREAD

**Issue by relatedcode** [relatedcode/ParseAlternatives#32]

**Comment by jhonkaman** [comment]
Has anyone tried migrating from Parse to Buddy? Or, is anyone familiar with Buddy in general? I'm considering migrating my Parse app to it but would like to hear from others before I make the move.

**Comment by jeacott1** [comment]
Thanks @Vietvantue,
  to clarify, Buddy has two offerings. The Buddy Platform offering is focused on Internet-of-Things projects. The newer Parse on Buddy offering is managed open-source Parse Server

Full disclosure, I work at buddy, we are a publicly listed company on the ASX, with global offices, and are fully committed to providing a first class, fully managed parse service for anyone who needs it for the long term. 
We are mirroring parse pricing, and running a scalable, full HA everything setup with full regular backups, monitoring etc for all apps, full customer disclosure of status. http://status.buddy.com , and plenty of available support where needed.

hope this helps.

**Comment by andresdouglas** [comment]
@jeacott1 Do you make the URL of the database public, so it can be accessed from our own server, for example?

**Comment by emjeykm** [comment]
not good

**Comment by jeacott1** [comment]
@andresdouglas, no parse-on-buddy does not make the database uri public. We have changed our pricing model since launch and are now are a free parse hosting service. We are not a free mongodb hosting service.
Allowing external access like this could compromise the performance, stability, and security of your application. 
parse-on-buddy provides a complete parse solution where your data is safe, regularly backed up, with application performance monitored and managed, and you can take a copy of your live database anytime you like directly from the dashboard.

**Comment by ndastur** [comment]
@jeacott1 I agree that public db access would compromise security. But some simple import export functions e.g. to json would be really useful

**Comment by jeacott1** [comment]
@ndastur as I stated in my previous comment, parse-on-buddy does provide export of your database via the dashboard and you are free to access that feature anytime you care to.

if you have an account, the export feature lives in the dashboard:
Navigate to 
App Settings/General
scroll down to the 'App Management' section and click the 'export database' button.

**Comment by ndastur** [comment]
@jeacott1 I'll have a look again but I am fairly sure it does a binary only export. I meant an export that was more generic like plain json. I also mention **import**. Which I am not aware exists. At the moment it is relatively difficult to even migrate one Buddy dataset to another

**Comment by jeacott1** [comment]
@ndastur the export creates a bson mongodump and gives you a URL to download it. 
if you want a more generic plain json you could pull directly from the parse api, or you could just convert the bson to json locally with bsondump https://docs.mongodb.com/manual/reference/program/bsondump/

for f in backups/*.bson; do bsondump "$f" > "$f.json"; done

re your import question - again "you can also import a database from the dashboard" - read my previous comment for details.
